server/modules/aws-notifier.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
```

Comment:
what does `modules` folder is about? is it a new folder ? can we move this to lib or helpers folder?